docs/pages/guides/monorepos.mdx

```diff
@@ -282,6 +294,30 @@ export default function App() {
 
 As mentioned earlier, using monorepos is not for everyone. You take on increased complexity and need to solve issues you most likely will run into. Here are a couple of common issues you might encounter.
 
+### Multiple React Native versions within the monorepo
+
+Expo SDK 50 and higher has improved support for more complext `node_modules` patterns, such as isolated modules. Unfortunately, React Native itself can still cause issues when installing multiple versions inside a single monorepo. Because of that, it's recommended to only use a single version of React Native.
```

Comment:
```suggestion
Expo SDK 50 and higher has improved support for more complete **node_modules** patterns, such as isolated modules. Unfortunately, React Native can still cause issues when installing multiple versions inside a single monorepo. Because of that, it's recommended to only use a single version of React Native.
```